src/main/java/com/zaxxer/sansorm/internal/Introspected.java

```diff
@@ -428,47 +406,56 @@ public String getTableName()
       return tableName;
    }
 
+   /**
+    * CLARIFY Must be public?
+    */
    public String getColumnNameForProperty(String propertyName)
    {
       for (FieldColumnInfo fcInfo : columnToField.values()) {
-         if (fcInfo.field.getName().equalsIgnoreCase(propertyName)) {
-            return fcInfo.columnName;
+         if (fcInfo.getPropertyName().equals(propertyName)) {
+            return fcInfo.getDelimitedColumnName();
          }
```

Comment:
Comparison is now case sensitive to be consistent with other public methods of this class which assume names to be case sensitive as provided in name element or property name.